### DISCUSSION TITLE
Hi ,Team,, content Progress is not happening on PDF ,

### DISCUSSION THREAD

**Discussion by sohailamjad12** [Sunbird-Ed/Community#612] [General Discussion]
will share the scenario here like when any students is mid of consuming courses if internet goes off the previous answered questions will not be recalled..in this students need again need to start from beginning or page 1..

**Comment by rajeevsathish** [comment]
@sohailamjad12 can you please be more clear what is the use-case and what is the problem you are trying to solve?

**Reply by Krishnaj20** [reply]
@sohailamjad12 - waiting for you to provide the necessary details for community to be able to help you.

**Reply by sohailamjad12** [reply]
Sure @Krishnaj20  will come on this  with proper scenario..Thanks

**Reply by Krishnaj20** [reply]
In the current implementation of ED, the resume for questions will always resume from the beginning.

**Reply by Krishnaj20** [reply]
@sohailamjad12 - Quick reminder that we are awaiting updates from your side.

**Reply by sohailamjad12** [reply]
Hi @Krishnaj20  as of now we are working on  some other modules priority release work..didn't check this issue and scenario ..we can close this issue for time being..Thanks for you help
